src/compute-pr-actions.ts

```diff
@@ -40,6 +40,7 @@ export interface Actions {
     shouldUpdateLabels: boolean;
     shouldUpdateProjectColumn: boolean;
     shouldRemoveFromActiveColumns: boolean;
+    isClassedAsReadyForMerge: boolean

```

Comment:
I assume prettier is going to add a semicolon here to match the other lines (or perhaps...remove the others?)

Reply:
```suggestion
    isMergeable: boolean;
```

?

“classed as” feels very clunky

Reply:
Nah, I've not set up prettier for this repo - some of the older code would probably look pretty bad so I didn't add it after I took over